### ISSUE TITLE
FR: Server description

### ISSUE THREAD

**Issue by baur** [phpservermon/phpservermon#623]
**Describe the feature you'd like**
Add for each address a short description as the server purpose etc, so that it is attached to the email notification

**Describe the benefits this feature would bring to PHPServermon users**
You can easily get lost when using **PHPServermon** to monitor (ping) hundreds of servers

**Additional context**
to push description in the name of server is not an option.
.

**Comment by damiankw** [comment]
I agree a description would be good, but what do you name your endpoints that you can't easily see what they are just from the Label? I mean, all of your servers should have a unique name and you should know what each of them do at a glance, unless your naming scheme is by serial number or something?

I'll have a look at this over the weekend if no one else gets to it by then.

**Comment by baur** [comment]
Updated to 3.4 (current Master), but I can't see this feature, this feature is very important for me (((

**Comment by damiankw** [comment]
The changes weren't committed, they were canned.

**Comment by baur** [comment]
@TimZ99 
Please merge this [PR](https://github.com/phpservermon/phpservermon/pull/665)
